packages/nuxt/schematics/migrations/4_0_0/snapshots/complex/after.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+const pending = 'something-else'
```

Comment:
Might also be worth handling the situation of:

```suggestion
const status = 'something-else'
const pending = 'something-else'
```